frame/1m/packm/bli_packm_cntl.h

```diff
@@ -33,67 +33,224 @@
 
 */
 
-struct packm_params_s
-{
-	uint64_t  size; // size field must be present and come first.
-	bszid_t   bmid_m;
-	bszid_t   bmid_n;
-	bool      does_invert_diag;
-	bool      rev_iter_if_upper;
-	bool      rev_iter_if_lower;
-	pack_t    pack_schema;
-	packbuf_t pack_buf_type;
+
+struct packm_cntl_s
+{
+	cntl_t        cntl; // cntl field must be present and come first.
+	packm_var_oft var;
+	const void*   params;
 };
-typedef struct packm_params_s packm_params_t;
+typedef struct packm_cntl_s packm_cntl_t;
+
+// -----------------------------------------------------------------------------
+
+BLIS_INLINE packm_var_oft bli_packm_cntl_variant( const cntl_t* cntl )
```

Comment:
@devinamatthews Do you have any objection to me renaming these functions from `variant` to `var`?

Reply:
No.

Reply:
Okay, thanks. (I'm still undecided.)